docs/swagger.yaml

```diff
@@ -1854,6 +1854,10 @@ paths:
                 Returned only if the "student_view_data" input parameter contains
                 this block's type.
 
+              * index_dictionary: (dict) The index_dictionary JSON data for this block.
```

Comment:
Should we say _what_ `index_dictionary` actually is? Something like "(usually this is the text content of the block, for search indexing or other purposes)"

Reply:
@bradenmacdonald Ah, yes. Updated! Thanks.